export_report.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/python3
+
+import plotly.io as pio
+import pandas as pd
+from docx import Document
+from docx.shared import Inches
+import os
+import re
+import sys
+import datetime
+
+def generateGraphs():
+    graphs = {}
+    for root, _, files in os.walk("."):
+        for f in files:
+            p = re.compile(r'[\w-]+_([\w-]+).(\w.+)')
+            m = p.match(f)
+            if 'reports' not in root and p.match(f) and m.group(2) == 'json':
+                df = pd.read_json(root+'/'+f, lines = True)
+
+                data = [dict(
+                type = 'scatter',
+                x = df['timestamp'],
+                y = df['latency']/1000000,
+                mode = 'markers',
+                transforms = [dict(
+                    type = 'groupby',
+                    groups = df['code']
+                )]
+                )]
+
+                layout = dict(
```

Comment:
Can you use the `{}` style of dictionary declaration here instead of the dict() function?